resend/webhooks/_webhooks.py

```diff
@@ -0,0 +1,251 @@
+        """
+
+    class DeleteWebhookResponse(TypedDict):
+        """
+        DeleteWebhookResponse is the type that wraps the response of the webhook that was deleted
+
+        Attributes:
+            object (str): The object type, always "webhook"
+            id (str): The ID of the deleted webhook
+            deleted (bool): Whether the webhook was successfully deleted
+        """
+
+        object: str
+        """
+        The object type, always "webhook"
+        """
+        id: str
+        """
+        The ID of the deleted webhook
+        """
+        deleted: bool
+        """
+        Whether the webhook was successfully deleted
+        """
+
+    @classmethod
+    def create(cls, params: CreateParams) -> CreateWebhookResponse:
+        """
+        Create a webhook to receive real-time notifications about email events.
+        see more: https://resend.com/docs/api-reference/webhooks/create-webhook
+
+        Args:
+            params (CreateParams): The webhook creation parameters
+                - endpoint: The URL where webhook events will be sent
+                - events: Array of event types to subscribe to
+
+        Returns:
+            CreateWebhookResponse: The created webhook response with id and signing_secret
+        """
+        path = "/webhooks"
+        resp = request.Request[Webhooks.CreateWebhookResponse](
+            path=path, params=cast(Dict[Any, Any], params), verb="post"
+        ).perform_with_content()
+        return resp
+
+    @classmethod
+    def get(cls, webhook_id: str) -> Webhook:
+        """
+        Retrieve a single webhook for the authenticated user.
+        see more: https://resend.com/docs/api-reference/webhooks/get-webhook
+
+        Args:
+            webhook_id (str): The webhook ID
+
+        Returns:
+            Webhook: The webhook object
+        """
+        path = f"/webhooks/{webhook_id}"
+        resp = request.Request[Webhook](
+            path=path, params={}, verb="get"
+        ).perform_with_content()
+        return resp
+
+    @classmethod
+    def update(cls, params: UpdateParams) -> UpdateWebhookResponse:
+        """
+        Update an existing webhook configuration.
+        see more: https://resend.com/docs/api-reference/webhooks/update-webhook
+
+        Args:
+            params (UpdateParams): The webhook update parameters
+                - webhook_id: The webhook ID
+                - endpoint: (Optional) The URL where webhook events will be sent
+                - events: (Optional) Array of event types to subscribe to
+                - status: (Optional) The webhook status ("enabled" or "disabled")
+
+        Returns:
+            UpdateWebhookResponse: The updated webhook response with id
+        """
+        webhook_id = params["webhook_id"]
+        path = f"/webhooks/{webhook_id}"
+
+        # Remove webhook_id from params before sending to API
+        update_payload = {k: v for k, v in params.items() if k != "webhook_id"}
```

Comment:
Do we really need to remove this? It seems like we don't do this in other methods like the domains one, and I'm fairly sure that looking at the code our API will just ignore this field.